### PR TITLE
[Console] fixed typo in phpdoc

### PR DESCRIPTION
**Pull request by nicolas-grekas**
| Q             | A
| ------------- | ---
| Branch?       | 2.7
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | -
| License       | MIT
| Doc PR        | -

### CODE REVIEW

**Review by Simperfit — APPROVED** [review]

**Comment by fabpot** [comment]
This change is wrong.

File: src/Symfony/Component/Console/Command/Command.php [hunk 1]
```diff
@@ -160,7 +160,7 @@ protected function interact(InputInterface $input, OutputInterface $output)
     }
 
     /**
-     * Initializes the command just after the input has been validated.
+     * Initializes the command before the input is validated.
```
Comment (chalasr) [inline src/Symfony/Component/Console/Command/Command.php]: Actually most part of the input validation happens in `Input::bind()` before that  `initialize()` is called, `Input:: validate()`. I would reword it relying on method names

**Comment by nicolas-grekas** [comment]
OK, I'm closing not need to spend more time here :)
